app/javascript/components/map/MapPopup.vue

```diff
@@ -8,10 +8,6 @@
       <BarChart :chartData="data" :options="options" :key="randomKey" />
       <MapLegend :data="data.datasets" />
     </div>
-    <div class="map__info-box">
-      <span class="map__info-box-icon"></span>
-        {{ this.text }}
```

Comment:
I think this can be removed from `data` and the `getChartData` method now

Reply:
done